Jenkinsfile

```diff
@@ -3189,7 +3189,7 @@ assert_frame_equal(training_curve, gt_curve, rtol=1e-3, atol=1e-3)"'''
         trainer.accelerator=gpu \
         trainer.log_every_n_steps=1 \
         trainer.val_check_interval=2 \
-        trainer.limit_val_batches=2 \
+        trainer.limit_val_batches=1 \
```

Comment:
can we increase this back to a higher number?
would be good to ensure we have diversity of values that check most cases?

Reply:
Yes ofc, I decreased it back to what it was originally (before PTL 2.0 upgrade) to reduce the time spent on CI tests since we now have limit_val_batches = 1 working okay (with it being made a multiple of num_microbatches inside our code). One caveat with increasing it back to higher number would be the CI run might take longer especially in cases where the default limit_val_batches is 50 in configs since we now do 50 full validation_steps instead of say 50//num_microbatches. Maybe in cases where limit_val_batches is high like 50, do limit_val_batches = 10 or something to save time on CI?

Reply:
This has been addressed as well.

Reply:
thanks!